src-mobile/script/components/checkTrackTextAndAddCarroussel.js

```diff
@@ -0,0 +1,29 @@
+import { putFade } from "./putFade.js";
```

Comment:
this function is redundant, the `checkPlaybackbarTextAndAddCarrousel` have the same purpose

Reply:
The checkPlaybackbarTextAndAddCarrousel function is designed to verify the text size and add a carousel, as its name suggests. In putFade, the function's objective is to add a fade effect based on the carousel's animation duration.

Reply:
The playback bar and track sections have different behaviors. For example, the playback bar should be dynamic, rendering and un-rendering as needed. Unfortunately, we are unable to combine them into a single function.

Reply:
Fixed whitin the next commit